package.json

```diff
@@ -18,6 +18,7 @@
   "activationEvents": [
     "onLanguage:plaintext",
     "onLanguage:markdown",
+    "onLanguage:markdown react",
```

Comment:
Is there a space in language name?

Reply:
I believe so. That is what it shows in VSCode:

![image](https://user-images.githubusercontent.com/1578436/76415278-4b0ec400-636f-11ea-8ddf-b90c746e0ae7.png)

@JDansercoer Since you opened the original issue, can you confirm that MDX files are considered "markdown react" in your editor as well? Establishing this will help ensure it is not merely my editor's setup/settings.

Reply:
I think it should be `mdx`.

Reply:
Good catch. I've updated it to be `mdx` instead of `Markdown React`.